### ISSUE TITLE
Are you using this bot?

### ISSUE THREAD

**Issue by Amaimersion** [Amaimersion/yandex-disk-telegram-bot#21]
At the moment I can't pay for VPS and domain name because Visa and MasterCard suspended their operations in Russia. So to understand if I should spend time on this project I need your feedback. Please answer in this issue if you using this bot, ideally with your usage scenario to understand expected load. Thank you!

**Comment by Bekzat-ainabek** [comment]
Yes, I have always used a bot. because, I don't have unlimited gigabytes. so I had to save money. Telegram has unlimited data. it would be very cool and convenient if you restore the bot

**Comment by bolshukhin** [comment]
Hello. I can provide for you own domain name & wsl 2 ( Ubuntu 20.04 ).

**Comment by RMirsaitov** [comment]
Привет. Твой бот очень нужен, аналогов не нашёл. Если хочешь, введи подписку минимальную на оплату, я бы платил небольшую сумму)

**Comment by adufftpc** [comment]
Бот нужОн!

**Comment by dorkygirl** [comment]
Yes pls I want to try out the bot
